### PR TITLE
chore: update prompt and amenity dataset

### PR DESCRIPTION
**Pull request by Yuan325**
- update list flight description to make date as required. (or it will throw an exception)
- update one of the amenities name

### CODE REVIEW

File: data/amenity_dataset.csv [hunk 1]
```diff
@@ -51,7 +51,7 @@ id,name,description,location,terminal,category,hour,sunday_start_hour,sunday_end
...
...75397873, -0.0005243446212261915, 0.0771387368440628, -0.033010635524988174, 0.009262710809707642, -0.003986199852079153, 0.023339958861470222, -0.020478054881095886, 0.02500850334763527, 0.0161531213670969, -0.058264464139938354, -0.04372072219848633, -0.004670826718211174, -0.021924549713730812, -0.04302164912223816, 0.031377073377370834, 0.024305559694767, -0.07181254029273987, -0.037175945937633514, 0.06264472752809525, 0.008027239702641964, 0.012945991940796375, 0.012187695130705833, 0.06361240893602371, -0.016734102740883827, 0.03563887998461723, 0.04209417849779129, -0.010218136943876743, -0.030389171093702316, -0.013068145141005516, -0.028042394667863846, 0.011199962347745895, 0.026231149211525917, -0.010196305811405182, -0.00928921066224575, -0.023917442187666893, -0.05183575674891472, -0.08772368729114532, -0.026064814999699593, 0.02156386896967888, 0.07809898257255554, -0.009879407472908497, 0.10865426063537598, 0.048208996653556824, 0.035701826214790344, -0.006512729451060295, 0.070896215736866, 0.020739324390888214, -0.020575588569045067, -0.01016781385987997, 0.016970185562968254, 0.007054878864437342, -0.04195088520646095, 0.02013476938009262, -0.007640668656677008]"
```
Comment (averikitsch) [inline data/amenity_dataset.csv:54]: Is this change expected?

File: llm_demo/orchestrator/vertexai_function_calling/functions.py [hunk 2]
```diff
@@ -22,7 +22,11 @@
 
 search_airports_func = generative_models.FunctionDeclaration(
     name="airports_search",
-    description="Use this tool to list all airports matching search criteria. Takes at least one of country, city, name, or all of the above criteria. This function could also be used to search for airport information such as iata code.",
+    description="""
```
Comment (averikitsch) [inline llm_demo/orchestrator/vertexai_function_calling/functions.py:25]: In the future, we might want to deduplicate these instructions across the orchestration modules

File: llm_demo/orchestrator/vertexai_function_calling/functions.py [hunk 3]
```diff
@@ -38,7 +42,16 @@
 
 search_amenities_func = generative_models.FunctionDeclaration(
     name="amenities_search",
-    description="Use this tool to search amenities by name or to recommend airport amenities at SFO. If top_k is not specified, default to 5",
+    description="""
+                Use this tool to search amenities by name or to recommended airport amenities at SFO.
+                If user provides flight info, use 'Get Flight' and 'Get Flights by Number'
```
Comment (averikitsch) [inline llm_demo/orchestrator/vertexai_function_calling/functions.py:47]: Are these the right tool names? Do we provide examples similar to the langchain tools?

**Review by averikitsch — APPROVED** [review]

File: llm_demo/orchestrator/vertexai_function_calling/functions.py (hunk 2 above)
Comment (Yuan325) [reply llm_demo/orchestrator/vertexai_function_calling/functions.py:25]: Yes, agree~ We can definitely dedup the tools / functions. I'll add this to the backlog :)

File: llm_demo/orchestrator/vertexai_function_calling/functions.py (hunk 3 above)
Comment (Yuan325) [reply llm_demo/orchestrator/vertexai_function_calling/functions.py]: Thanks for catching this. I update these tool names. Checked langchain tools, it is updated there as well and it have the right name.

**Pull request merged**